drivers/include/net/netdev.h

```diff
@@ -54,6 +54,7 @@ extern "C" {
 #include <sys/uio.h>
 
 #include "net/netstats.h"
+#include "net/l2filter.h"
```

Comment:
We do not use it in other modules where such constructs exist (e.g. `netstats.h`) so I wouldn't do it here either. If you are not happy with that, please provide a follow-up for these cases and let's discuss it there.